conf/httpd.conf.d/httpd.admin

```diff
@@ -103,6 +103,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
 $User = "pf";
 $Group = "pf";
 
+$PerlChildInitHandler = "pf::WebAPI::ChildInitHandler";
```

Comment:
Done.

--  
LM


On Wednesday, March 25, 2015 at 10:12 , jrouzierinverse wrote:

> In conf/httpd.conf.d/httpd.admin (https://github.com/inverse-inc/packetfence/pull/431#discussion_r27123373):
> > @@ -103,6 +103,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '') > $User = "pf"; > $Group = "pf"; > > +$PerlChildInitHandler = "pf::WebAPI::ChildInitHandler";  
> Also PerlChildInitHandler pf::WebAPI::InitHandler::child_init was removed from httpd.admin.conf
> It was needed because of the non-random password generation  
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/inverse-inc/packetfence/pull/431/files#r27123373).
>  
>  
>  
>  
>